File: Solutions/IQCare.Core/IQCare.PMTCT.Core/Models/HEI/PatientIptOutcome.cs
using System;

namespace IQCare.PMTCT.Core.Models.HEI
{
    public class PatientIptOutcome
    {
        public int Id { get; set; }
        public int PatientMasterVisitId { get; set; }
        public int PatientId { get; set; }
        public bool IptEvent { get; set; }
        public string ReasonForDiscontinuation { get; set; }
        public bool DeleteFlag { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreateDate { get; set; }
    }
}

File: Solutions/IQCare.Core/IQCare/frontend/app/pmtct/_models/hei/PatientIptOutcome.ts
export interface PatientIptOutcome {
    Id?: number;
    PatientMasterVisitId?: number;
    PatientId?: number;
    IptEvent?: boolean;
    ReasonForDiscontinuation?: string;
    DeleteFlag?: boolean;
    CreatedBy?: number;
    CreateDate?: Date;
}
